dash-renderer/.eslintrc.json

```diff
@@ -1,5 +1,28 @@
 {
     "extends": ["eslint:recommended", "prettier"],
+    "settings": {
+      "react": {
+        "createClass": "createReactClass", // Regex for Component Factory to use,
+                                           // default to "createReactClass"
+        "pragma": "React",  // Pragma to use, default to "React"
+        "version": "detect", // React version. "detect" automatically picks the version you have installed.
+                             // You can also use `16.0`, `16.3`, etc, if you want to override the detected value.
+                             // default to latest and warns if missing
+                             // It will default to "detect" in the future
+        "flowVersion": "0.53" // Flow version
```

Comment:
I don't think we use flow at all in the renderer

Reply:
this was a blind copy from [here](https://github.com/yannickcr/eslint-plugin-react#configuration) after a warning raised in eslint

I can remove it